website/docs/r/log_analytics_datasource_iis_logs.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Log Analytics"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_log_analytics_datasource_iis_logs"
+description: |-
+  Manages a Log Analytics IIS Logs DataSource.
+---
+
+# azurerm_log_analytics_datasource_iis_logs
+
+Manages a Log Analytics IIS Logs DataSource.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "example-law"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_log_analytics_datasource_iis_logs" "example" {
+  name                = "example-lad-wpc"
+  resource_group_name = azurerm_resource_group.example.name
+  workspace_name      = azurerm_log_analytics_workspace.example.name
+  state               = "OnPremiseEnabled"
```

Comment:
Can we update this to reflect the new value per the comment above? 
```suggestion
  on_premise               = true
```

Reply:
Resolved to reflect new schema properties